nomad/structs/structs.go

```diff
@@ -2514,6 +2514,17 @@ var (
 	}
 )
 
+var (
+	defaultServiceJobReschedulePolicy = ReschedulePolicy{
```

Comment:
It is generally useful to export these so you can use them in the api pkg when setting up the defaults.